### DISCUSSION TITLE
Import Action error

### DISCUSSION THREAD

**Discussion by sasori13L** [filamentphp/filament#11832] [Help] [answered]
### Package

Actions

### Package Version

v3.2.0

### How can we help you?

When using import action causes error.
SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'file_path' at row 1

![image](https://github.com/filamentphp/filament/assets/16425810/238691f4-018d-4472-badb-e35b6a96e139)

**Comment by leandrocfe** [comment, marked as answer]
maybe changing the column type to `longtext`?
